lib/schema.js

```diff
@@ -135,97 +55,117 @@ class Schema {
     this.indexes = {};
     this.references = new Set();
     this.relations = new Set();
-    this.validate = defs.validate || null;
-    this.format = defs.format || null;
-    this.parse = defs.parse || null;
-    this.serialize = defs.serialize || null;
-    this.preprocess(defs);
+    this.validate = setValidate(raw.validate);
+    this.format = raw.format || (() => {});
+    this.parse = raw.parse || (() => {});
+    this.serialize = raw.serialize || (() => {});
```

Comment:
We don't need additional round parentheses around functions here

Reply:
I thought it can be better to keep here functions, just to avoid `null` and avoid more checks for future development. So it is similar, that I did with `validate` there is no reason to check if `validate === null (!validate)`. So, goal here - to reduce amount of checks and if statements.

Reply:
I am using metarhia eslint config and this parentheses issue, because of lint error

Reply:
IMO we shouldn't have format/parse/serialize as "default" functions. Because this is optional behavior and with such approach it becomes not clear how to distinguigsh custom `serialize` vs `do nothing`.